addons/optionsmenu/stringtable.xml

```diff
@@ -345,6 +345,22 @@
             <German>Protokolliert Debug-Informationen im RPT und speichert sie in der Zwischenablage.</German>
             <Portuguese>Envia informação de depuração para RPT e área de transferência.</Portuguese>
         </Key>
+		<Key ID="STR_ACE_OptionsMenu_headBugFix">
+            <English>Headbug Fix</English>
```

Comment:
Don't add all languages if the values aren't translated. It's hard to find those instances for translators.